src/main/java/com/selfxdsd/todocli/JsonTodo.java

```diff
@@ -0,0 +1,28 @@
+package com.selfxdsd.todocli;
+
+import javax.json.Json;
+
+/**
+ * Todo as JsonObject.
+ *
+ * @author lukacupic
+ */
+public class JsonTodo extends AbstractJsonObject {
+
+    /**
+     * Ctor.
+     *
+     * @param todo Todo to be converted to JsonObject.
+     */
+    public JsonTodo(final Todo todo) {
+        super(Json.createObjectBuilder()
+                // @todo #4:5min Uncomment the line below once #10 is merged
+//                .add("id", todo.getID())
```

Comment:
@lukacupic I think checkstyle complains about this comment. Just remove this line and write the todo in the JavaDoc of the method. "Add id attribute when #10 is finished"